docs/en/02_Developer_Guides/09_Security/04_Secure_Coding.md

```diff
@@ -585,6 +583,80 @@ In a future release this behaviour will be changed to be on by default, and this
+
+In addition to this ensure the users are provided with secure cookies by setting the recommended "Secure" and 
+"HTTPOnly" flags. These flags prevent them from being stolen by an attacker through javascript. 
+
+ - The "Secure" cookie flag instructs the browser not to send the cookie over an insecure HTTP connection. If this 
+flag is not present, the browser will send the cookie even if HTTPS is not in use, which means it is transmitted in 
+clear text and can be intercepted and stolen by an attacker who is listening on the network.
+
+- The "HTTPOnly" flag lets the browser know whether or not a cookie should be accessible by client-side JavaScript 
+code. It is best practice to set this flag unless the application is known to use JavaScript to access these cookies 
+as this prevents an attacker who achieves cross-site scripting from accessing these cookies.
+
+
+	:::php
+	
+	Cookie::set('cookie-name', 'chocolate-chip', $expiry = 30, $path = null, $domain = null, $secure = true, 
+		$httpOnly = false
+	);
+
+## Security Headers
+
+In addition to forcing HTTPS browsers can support additional security headers which can only allow access to a website 
+via a secure connection. As browsers increasingly provide negative feedback regarding unencrypted HTTP connections, 
+ensuring an HTTPS connection will provide a better and more secure user experience.  
+
+- The `Strict-Transport-Security` header instructs the browser to record that the website and assets on that website 
+MUST use a secure connection. This prevents websites from becoming insecure in the future from stray absolute links 
+or references without https from external sites. Check if your browser supports [HSTS](https://hsts.badssl.com/)
+  - `max-age` can be configured to anything in seconds: `max-age=31536000` (1 year), for roll out, consider something 
+  lower
+  - `includeSubDomains` to ensure all present and future sub domains will also be HTTPS
+
+For sensitive pages, such as members areas, or places where sensitive information is present, adding cache control
+ headers can explicitly instruct browsers not to keep a local cached copy of content and can prevent content from 
+ being cached throughout the infrastructure (e.g. Proxy, caching layers, WAF etc). 
+ 
+- The headers `Cache-control: no-store` and `Pragma: no-cache` along with expiry headers of `Expires: <current date>` 
+and `Date: <current date>` will ensure that sensitive content is not stored locally or able to be retrieved by 
+unauthorised local persons. SilverStripe adds the current date for every request, and we can add the other cache 
+ headers to the request for our secure controllers:
+
+
+	:::php
+
+	class MySecureController extends Controller {
+		
+		public function init() {
+			parent::init();
+        	
+			// Add cache headers to ensure sensitive content isn't cached.
+			$this->response->addHeader('Cache-control', 'max-age=0, must-revalidate, no-transform');
```

Comment:
Hey @tractorcow
Good catch, i've fixed the case problem.
 `no-cache` - Forces caches to submit the request to the origin server for validation before releasing a cached copy.
`max-age` - Specifies the maximum amount of time a resource will be considered fresh. Contrary to Expires, this directive is relative to the time of the request. Max age is generally used for assets and is relative to the  but i'm following best practices recommended from a security review.

In theory no-cache should be enough (but that's being added by the `add_cache_headers` call now. The other headers just define more about the request to ensure nothing is reused, and everything should be re-requested, and some behaviour at WAF's vary depending on other headers used. By including them all (including pragma) I'm trying to cover all HTTP versions and providers and possible platforms.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control
https://www.owasp.org/index.php/HTTP_Strict_Transport_Security_Cheat_Sheet